applications/StructuralMechanicsApplication/custom_advanced_constitutive/generic_anisotropic_3d_law.cpp

```diff
@@ -0,0 +1,669 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:         BSD License
+//                   license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Alejandro Cornejo 
+//  Collaborator:    Lucia Barbu
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "utilities/math_utils.h"
+#include "structural_mechanics_application_variables.h"
+#include "generic_anisotropic_3d_law.h"
+#include "custom_utilities/tangent_operator_calculator_utility.h"
+
+
+namespace Kratos
+{
+ConstitutiveLaw::Pointer GenericAnisotropic3DLaw::Create(Kratos::Parameters NewParameters) const
+{
+    return Kratos::make_shared<GenericAnisotropic3DLaw>();
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+void GenericAnisotropic3DLaw::CalculateMaterialResponsePK1(ConstitutiveLaw::Parameters& rValues)
+{
+    this->CalculateMaterialResponsePK2(rValues);
+
+    Vector& stress_vector                = rValues.GetStressVector();
```

Comment:
```suggestion
    Vector& r_stress_vector                = rValues.GetStressVector();
```

Idem for the other references

Reply:
done